presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/glue/converter/GlueToPrestoConverter.java

```diff
@@ -89,7 +89,6 @@ public static Table convertTable(com.amazonaws.services.glue.model.Table glueTab
             // the table is an Iceberg/Delta table and decide whether to redirect or fail.
             tableBuilder.setDataColumns(ImmutableList.of(new Column("dummy", HIVE_INT, Optional.empty(), Optional.empty())));
             tableBuilder.getStorageBuilder().setStorageFormat(StorageFormat.fromHiveStorageFormat(HiveStorageFormat.PARQUET));
-            tableBuilder.getStorageBuilder().setLocation(sd.getLocation());
```

Comment:
instead of all these changes, could we have just set the location to an empty string?  I see other places in the code that check for whether this field is an empty string.

Reply:
Thank you for proposing this, now it is a much more simple fix.